js/bootstrap-slider.js

```diff
@@ -1371,11 +1371,16 @@
 				var sliderOffset = this._state.offset[this.stylePos];
 				var distanceToSlide = eventPosition - sliderOffset;
 				// Calculate what percent of the length the slider handle has slid
-				var percentage = (distanceToSlide / this._state.size) * 100;
-				percentage = Math.round(percentage / this._state.percentage[2]) * this._state.percentage[2];
-				if (this.options.reversed) {
-					percentage = 100 - percentage;
-				}
+				var selectedPercentage = (distanceToSlide / this._state.size) * 100;
+		                var nearestPercentage = Math.round(selectedPercentage / this._state.percentage[2]) * this._state.percentage[2];
+		
+		                // if the maximum is not equal to a step value, select max when user selects the max value.
+		                var percentage = 100 - selectedPercentage < Math.abs(nearestPercentage - selectedPercentage) ?
```

Comment:
could you refactor the boolean `100 - selectedPercentage < Math.abs(nearestPercentage - selectedPercentage)` into a variable describing what the condition is, and then reference it here?